test/bzz-protocol.spec.ts

```diff
@@ -289,15 +279,34 @@ describe('BZZ protocol', () => {
       }
 
       try {
-        await iframeWindow.window.swarm.localStorage.getItem(commonKeyName)
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        const swarm = iframeWindow.window.swarm
       } catch (e) {
-        if ((e as string).endsWith(`${commonKeyName} does not exist`)) return true
+        if (
+          e.stack &&
+          typeof e.stack === 'string' &&
+          e.stack.startsWith(`Error: Blocked a frame with origin "null" from accessing a cross-origin frame.`)
```

Comment:
This is not too nice, and also seems a bit unstable.

Is there no other way of checking the error? Does it not set a `code`, `message`, `errno` or any other property?